cypress.config.ts

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
this file has no reason to be in the root

Reply:
Where is this located in the tree? I can't seem to find this. 

It looks like Cypress put these files there. Maybe they should be in .gitignore so as not to contaminate future builds?